monkey/monkey_island/cc/ui/src/components/ui-components/plugins-marketplace/AvailablePlugins.tsx

```diff
@@ -56,39 +63,37 @@ const AvailablePlugins = (props) => {
     });
   }, [installedPlugins]);
 
-  const filterPlugins = () => {
-    let shownPlugins = availablePlugins;
+  const filterRows = () => {
+    let allRows = availablePluginRows;
     for (const filter of Object.values(filters)) {
-      shownPlugins = shownPlugins.filter(filter);
+      allRows = allRows.filter(filter);
     }
-    setDisplayedPlugins(shownPlugins);
+    setDisplayedRows(allRows);
   }
 
+  //TODO refactor this method
   const disableInstallAllSafePlugins = () => {
     let unSafeDispalyedPlugins = [];
     let safeDispalyedPlugins = [];
-    for (const plugin of displayedPlugins) {
+    for (const plugin of displayedRows) {
       if (!plugin.safe) {
         unSafeDispalyedPlugins.push(plugin.name);
       } else {
         safeDispalyedPlugins.push(plugin.name);
       }
     }
-    setInstallingAllSafePlugins(unSafeDispalyedPlugins.length > 0 && safeDispalyedPlugins.length === 0 || displayedPlugins.length === 0)
-  }
-
-  const onRefreshCallback = () => {
-    setSuccessfullyInstalledPluginsIds([]);
+    setInstallingAllSafePlugins(unSafeDispalyedPlugins.length > 0 && safeDispalyedPlugins.length === 0 || displayedRows.length === 0)
```

Comment:
Noticed this typo: `unSafeDispalyedPlugins` -> `unSafeDisplayedPlugins`

Reply:
There's a todo to refactor this method, I already have the changes, but out of scope